cmd/geth/chaincmd.go

```diff
@@ -165,6 +165,17 @@ It's deprecated, please use "geth db export" instead.
 		}, utils.DatabaseFlags),
 		Description: `
 This command dumps out the state for a given block (or latest, if none provided).
+`,
+	}
+	exportSnapshotPreimagesCommand = &cli.Command{
+		Action:    exportSnapshotPreimages,
+		Name:      "export-snapshot-preimages",
+		Usage:     "Export the preimage in snapshot enumeration order",
+		ArgsUsage: "<dumpfile>",
+		Flags:     flags.Merge([]cli.Flag{utils.TreeRootFlag}, utils.DatabaseFlags),
```

Comment:
You could do something like "<dumpfile> [<rootHash>] here, (similar to the the exportCommand), so we don't need to define another flag for the tree root